pyalgotrading/algobulls/connection.py

```diff
@@ -524,7 +533,7 @@ def start_job(self, strategy=None, start=None, end=None, instruments=None, lots=
         self.strategy_locale_map[trading_type][strategy] = location
         return response
 
-    def backtest(self, strategy=None, start=None, end=None, instruments=None, lots=1, parameters=None, candle=None, mode=StrategyMode.INTRADAY, delete_previous_trades=True, initial_funds_virtual=1e9, vendor_details=None, **kwargs):
+    def backtest(self, strategy=None, start=None, end=None, instruments=None, lots=None, parameters=None, candle=None, mode=None, delete_previous_trades=True, initial_funds_virtual=None, vendor_details=None, **kwargs):
```

Comment:
why have we removed the default values?

Reply:
while checking in sanity I can prioritize the saved parameters as well